ci-operator/config/redhat-developer/service-binding-operator/redhat-developer-service-binding-operator-master__4.13.yaml

```diff
@@ -50,7 +37,7 @@ tests:
         else
           OPERATOR_INDEX_IMAGE_REF=quay.io/redhat-developer/servicebinding-operator:pr-${PULL_NUMBER}-${PULL_PULL_SHA:0:8}-index;
         fi;
-        make -k VERBOSE=2 OPERATOR_INDEX_IMAGE_REF=$OPERATOR_INDEX_IMAGE_REF CATSRC_NAME=sbo-pr-checks SKIP_REGISTRY_LOGIN=true EXTRA_BEHAVE_ARGS="--tags=~@crdv1beta1 --tags=~@olm-descriptors --tags=~@upgrade-with-olm --tags=~@disable-openshift-4.9 --tags=~@disable-openshift-4.8+ --tags=~@disable-openshift-4.9+" -o registry-login test-acceptance-with-bundle test-acceptance-artifacts
+        make -k VERBOSE=2 OPERATOR_INDEX_IMAGE_REF=$OPERATOR_INDEX_IMAGE_REF CATSRC_NAME=sbo-pr-checks SKIP_REGISTRY_LOGIN=true EXTRA_BEHAVE_ARGS="--tags=~@crdv1beta1 --tags=~@olm-descriptors --tags=~@upgrade-with-olm --tags=~@disable-openshift-4.12 --tags=~@disable-openshift-4.13 --tags=~@disable-openshift-4.8+ --tags=~@disable-openshift-4.9+ --tags=~@disable-openshift-4.10+ --tags=~@disable-openshift-4.11+ --tags=~@disable-openshift-4.12+ --tags=~@disable-openshift-4.13+" -o registry-login test-acceptance-with-bundle test-acceptance-artifacts
```

Comment:
Are the `--tags=~@disable-openshift-4.13`  `--tags=~@disable-openshift-4.13+` needed here?

Reply:
It is not necessary in terms of the current set of acceptance tests for SBO, however this is part of the "testing framework" so that allows disabling scenarios for 4.13 in the future, if we need to.